app/errors.py

```diff
@@ -0,0 +1,14 @@
+class NotWearingMaskError(BaseException):
+    pass
+
+
+class VaccineError(Exception):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
```

Comment:
NotVaccinatedError and OutdatedVaccineError should have init() to use inherited init() from a base class:
Example:
class OutdatedVaccineError(VaccineError):
def init(self, message):
super.init(message)